test/dspal_tester/adsp_proc/gpio_test_imp.c

```diff
@@ -40,8 +40,9 @@
 #include "test_status.h"
 #include "test_utils.h"
 
-#define GPIO_DEVICE_PATH  "/dev/gpio-56"
-#define GPIO_INT_DEVICE_PATH  "/dev/gpio-55"
+#define GPIO_DEVICE_PATH  "/dev/gpio-10"
+#define GPIO_DEVICE_PATH_LOOPBACK  "/dev/gpio-11"
+#define GPIO_INT_DEVICE_PATH  "/dev/gpio-11"
 
```

Comment:
Lets move these to test/dspal_tester/adsp_proc/platform.h and localize ifdefs for ADSP and SLPI differences.